repository_service_tuf_api/api/bootstrap.py

```diff
@@ -28,8 +28,10 @@ def get():
     "/",
     summary="Bootstrap the system with initial signed Metadata.",
```

Comment:
I suggest we make it explicit in the summaries. Here my suggestion
```suggestion
    summary="Post a task to bootstrap the system with initial signed Metadata.",
```

Reply:
Thank you @kairoaraujo  for the review. I updated the endpoints with this standard. Let me know what you think.